evalml/tests/component_tests/test_components.py

```diff
@@ -1297,6 +1312,8 @@ def test_estimators_accept_all_kwargs(
         )
     if estimator_class.model_family == ModelFamily.ENSEMBLE:
         params = estimator.parameters
+    elif estimator_class.model_family == ModelFamily.PROPHET:
+        params = estimator.get_params()
```

Comment:
Would using `estimator.parameters` work for prophet?

Reply:
Yes that would work. I'm testing this in particular because Prophet has its own `_parameters` output that covers a wide range of parameters being passed into the component.